### ISSUE TITLE
How to transform highway lines into mesh?

### ISSUE THREAD

**Issue by WooDOd** [domlysz/BlenderGIS#718]
# **Describe the bug**

**I would like to know how to transform the lines of vertices, in a Mesh**
**I need your help as I need to finish a project**

# **How to Reproduce**

![image](https://user-images.githubusercontent.com/81538690/191145528-8c0a9dda-b01a-451d-bf44-2bbed467dbf8.png)

**how i wanted it to stay:**

![image](https://user-images.githubusercontent.com/81538690/191145687-b63f6782-8237-4eb9-bbee-3048599fd5c5.png)

**Comment by ea8885** [comment]
Have you figured this out? I been trying for a day now, no luck :(

**Comment by ea8885** [comment]
try extruding it.

**Comment by WooDOd** [comment]
Yes, find out, in the case was using the skin modifier, the streets get really cool using it. Watch this whole video to learn: https://youtu.be/RGULgi3f-1c

**Comment by m7d2** [comment]
> Yes, find out, in the case was using the skin modifier, the streets get really cool using it. Watch this whole video to learn: https://youtu.be/RGULgi3f-1c

It doesn't work
